server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -615,11 +628,22 @@ public void createAlignedTimeSeries(
       List<TSEncoding> encodings,
       List<CompressionType> compressors)
       throws MetadataException {
-    createAlignedTimeSeries(
+    createAlignedTimeSeriesEntry(
         new CreateAlignedTimeSeriesPlan(
             prefixPath, measurements, dataTypes, encodings, compressors, null));
   }
 
+  public void createAlignedTimeSeriesEntry(CreateAlignedTimeSeriesPlan plan)
+      throws MetadataException {
+    createAlignedTimeSeries(plan);
```

Comment:
rename this method

Reply:
renamed to createAlignedTimeSeriesAndCheckIDTable;